scripts/sns/install/upgrade_dapp.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script is used to create an SNS proposal for increasing the cost of proposal rejection to 200 MOD
+# This script is called by upgrade_test_canister.sh
+# For detailed explanation please see the GitHub Repo: https://github.com/dfinity/sns-testing/tree/main
+
+
+set -euo pipefail
+
+CURRENTDIR="$(pwd)"
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
+
+SCRIPT_DIR="$(pwd)"
+
+. ./constants.sh normal
+
+export DEVELOPER_NEURON_ID="194d3f742383afcee2f3fb4a1075aa6f9652f0299bc65cb3da353265206814b0"
+
+cd "${CURRENTDIR}"
+
+# See the proposal format at https://internetcomputer.org/docs/current/developer-docs/integrations/sns/managing/managing-nervous-system-parameters
+quill sns  \
+   --canister-ids-file "${SCRIPT_DIR}/sns_canister_ids.json" \
```

Comment:
${SCRIPT_DIR} is this correct ?

Reply:
`REPO_DIR="$SCRIPT_DIR/../../../"`

Reply:
yes must be, just replaced REPODIR with SCRIPT_DIR in two places

Reply:
it will point to  scripts/sns/install/update_sns.sh